demos/guide/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "guide"
+version = "0.1.0"
+authors = ["Zicklag <zicklag@katharostech.com>"]
+edition = "2018"
+
```

Comment:
oh, btw. we should add `publish = false` to make it explicit to the tools this won't be a binary that's gonna be published